src/main/scala/de/upb/cs/uc4/hyperledger/utilities/RegistrationManager.scala

```diff
@@ -54,13 +54,15 @@ object RegistrationManager extends RegistrationManagerTrait {
     )
   }
 
+  // TODO set affiliation?

```

Comment:
done? not done?

Reply:
Same thing here, affiliations not yet in certificates. Not sure if it will be automatically copied from registering admin, but cannot be set currently, because the network does not seem to know the affiliation yet.